File: Examples/VisualBasic/WorkingWithProjects/WorkingWithProjectProperties/WriteWeekdayProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports Aspose.Tasks.Saving

Namespace WorkingWithProjects.WorkingWithProjectProperties
    Public Class WriteWeekdayProperties
        Public Shared Sub Run()
            ' ExStart: WriteWeekdayProperties
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Create a project instance
            Dim project As New Project(dataDir & "WriteWeekdayProperties.mpp")

            ' Set week days properties
            project.Set(Prj.WeekStartDay, DayType.Monday)
            project.Set(Prj.DaysPerMonth, 24)
            project.Set(Prj.MinutesPerDay, 540)
            project.Set(Prj.MinutesPerWeek, 3240)
            project.Save(dataDir & "saved_out.xml", SaveFileFormat.XML)
            ' ExEnd: WriteWeekdayProperties
        End Sub
    End Class
End Namespace

File: Examples/CSharp/WorkingWithProjects/WorkingWithProjectProperties/WriteWeekdayProperties.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects.WorkingWithProjectProperties
{
    using Aspose.Tasks.Saving;

    internal class WriteWeekdayProperties
    {
        public static void Run()
        {
            //ExStart:WriteWeekdayProperties
            //ExFor: Prj.WeekStartDay
            //ExFor: Prj.DaysPerMonth
            //ExFor: Prj.MinutesPerDay
            //ExFor: Prj.MinutesPerWeek
            //ExSummary: Shows how to write project's weekday properties.

            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            // Create a project instance
            var project = new Project(dataDir + "WriteWeekdayProperties.mpp");

            // Set week days properties
            project.Set(Prj.WeekStartDay, DayType.Monday);
            project.Set(Prj.DaysPerMonth, 24);
            project.Set(Prj.MinutesPerDay, 540);
            project.Set(Prj.MinutesPerWeek, 3240);
            project.Save(dataDir + "WriteWeekdayProperties_out.xml", SaveFileFormat.XML);
            //ExEnd:WriteWeekdayProperties            
        }
    }
}